node/tessel-export.js

```diff
@@ -1399,7 +1418,25 @@ function getWifiInfo() {
                   if (error) {
                     reject(error);
                   } else {
-                    network.ips = ipResults.split('\n');
+                    network.ip = ipResults
+                      .split('\n') // split multiple line string into array of strings
+                      .filter((string) => string.includes('Bcast'))[0] // find the string containing a reference to "Bcast" (Broadcast IP)
+                      .trim() // remove extra whitespace from either side of the string
+                      .split(' ') // split the string by whitespace between words
+                      .filter((string) => string.includes('Bcast'))[0] // find the string containing a reference to "Bcast"
+                      .split(':')[1]; // split the string by the ":", i.e. "Bcast:10.0.0.11" and grab the number as the 2nd item in that array
+
+                    // attempt to parse out the security configuration from the returned network object
+                    if (network.encryption.enabled) {
+                      if (network.encryption.wep) {
+                        network.security = 'wep';
+                      } else if (network.encryption.authentication && network.encryption.wpa) {
+                        // sets "security" to either psk or psk2
+                        network.security = `${network.encryption.authentication[0]}${network.encryption.wpa[0] === 2 ? 2 : null}`;
```

Comment:
I don't believe it will return `"null"` if `network.encryption.wpa[0]` doesn't equal `2`, it shouldn't append any value to `network.encryption.authentication[0]`. There is a test to verify this => https://github.com/tessel/t2-firmware/pull/195/files#diff-4df274bad3cd8322e337b8eac6af233bR2115